examples/knowledge_tuning/instructlab/document_collection/ibm-annual-report/qna.yaml

```
@@ -0,0 +1,100 @@
+
+    · Lower gains on land/building dispositions ($126 million) included in "Other"; partially offset by
+
+    · A gain of $349 million from the sale of certain QRadar SaaS assets in 2024, included in "Other". Refer to note E, "Acquisitions & Divestitures," for additional information; and
+
+    · Higher gains on divestitures ($206 million) primarily driven by the divestiture of The Weather Company assets ($243 million), included in "Other". Refer to note E, "Acquisitions & Divestitures," for additional information; and
+
+    · Higher gains on sales of intangibles ($87 million) included in "Other"; and
+
+    · Higher interest income ($77 million) primarily driven by a higher average cash balance in the current year.
+  questions_and_answers:
+    - question: Why did IBM report a total "Other (income) and expense" of $1,871 million in 2024 compared to income of $914 million in 2023?
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)